scalableminds/webknossos
# Write wkurl into annotation nml file

### URL of deployed dev instance (used for testing):
- https://wkurlnml.webknossos.xyz/

### Steps to test:
- Create an annotation
- Upload the annotation from the dashboard -> no problem
- In your downloaded nml file, change the org (or dataset name) and wkUrl
- See a descriptive error

When uploading annotations from the dataset view it seems like org and dataset are not checked? Therefore wkUrl is not used there.

### Issues:
- fixes #6610 

------
